dockstore-webservice/src/main/java/io/dockstore/webservice/core/VersionMetadata.java

```diff
@@ -69,8 +74,16 @@ public class VersionMetadata {
 
     @Column()
     @Pattern(regexp = "10\\.[^/]++/.++")
+    @Deprecated(since = "1.16")
+    @Schema(deprecated = true)
     protected String doiURL;
 
+    @ManyToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
```

Comment:
Does it also work out to a limit of three here?

Reply:
Yeah

Reply:
> Could you please elaborate on https://github.com/dockstore/dockstore/pull/5879/files/6333c924f9561b0443c2134ca3993a16c58adaa1#r1612139110 with a concrete example maybe? Want to make sure I understand what we're trying to prevent...

With @coverbeck's GitHub DOI harvesting work, if a multi-workflow repository has a GitHub DOI then we are going to assign it to all workflows in the repository. 

Suppose that the DOI is 10.foo/bar and we have workflow A and workflow B that belong to the same repository. I believe the scenario we are trying to prevent is having duplicate DOI entities in our table, like two instances of DOI 10.foo/bar, one for workflow A and one for workflow B. We want a single DOI entity that is referenced by both workflow A and workflow B